maintainers/maintainer-list.nix

```diff
@@ -850,6 +850,12 @@
     githubId = 858965;
     name = "Andrew Morsillo";
   };
+  amz-x = {
+    email = "mail@amz-x.com";
+    github = "amz-x";
+    githubId = 18249234;
+    name = "Christopher Crouse";
+  };
```

Comment:
Hi, can you please [put this in a separate commit](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers) `maintainers: add amz-x` so there are exactly 2 commits in this PR?

Reply:
Hi, @bobby285271 updated the PR as per request.